unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowHUDController.cs

```diff
@@ -51,21 +75,22 @@ public void SendChatMessage(string msgBody)
 
         var data = new ChatMessage()
         {
-            body = msgBody,
+            body = $"/w {conversationUserName} " + msgBody,
             sender = UserProfile.GetOwnUserProfile().userId,
+            messageType = ChatMessage.Type.PRIVATE
         };
 
         WebInterface.SendChatMessage(data);
     }
 
     public void SetVisibility(bool visible)
     {
+        if (view.gameObject.activeSelf == visible) return;
+
         view.gameObject.SetActive(visible);
 
         if (visible)
-        {
             view.StartCoroutine(ForceLayoutDelayed());
```

Comment:
Please use `Utils.ForceUpdateLayout` here. Also, this should be under view's `OnEnable` as in `FriendsTabViewBase`.

Reply:
done